src/include/sof/audio/rtnr/rtklib/include/RTK_MA_API.h

```diff
@@ -44,6 +44,16 @@ int RTKMA_API_Set(void *Context, const void *pParameters, int size, unsigned int
 
 int	RTKMA_API_Get(void *Context, void *pParameters, int size, unsigned int IDs);
 
+int	RTKMA_API_Set_Data_1(void *Context, unsigned int *data,
+						unsigned int msg_index,
+						unsigned int num_elms,
+						unsigned int elems_remaining);
+
+int	RTKMA_API_Set_Data_2(void *Context, unsigned int *data,
+						unsigned int msg_index,
+						unsigned int num_elms,
+						unsigned int elems_remaining);
+
```

Comment:
When should we use 1 and when should we use 2 ? can the be better named so users can understand the usage better. I would also put some comments explaining the difference and usage.

Reply:
@cujomalainey  @lgirdwood 

Thanks for the review. We'll combine those 2 APIs into 1 to avoid confusion.

Another question is what's the best practice to send binary configuration file to sof component during system startup? The config is about 2KB, and should only be set once.

We've been using sof-ctl which works well, but the same configuration will be sent every time before recording. Besides, using sof-ctl might not be a proper way for final product.

We've also tried using cset-tlv to send config just like the control blobs for the UI. However the config is much larger than control blobs and I have no success during my experiments. Is there file size limit for this?

Thanks.

Reply:
Just force-pushed a commit to simplify the Set_Data API.

There could be different types of the data, but yes they should be handled internally in library.

Thanks!

Reply:
You could have a the config data as a static data block as part of the RTNT.  This would be build into the DATA section and could be used by default ?

Reply:
Yes currently there is a default config stored in the static table inside the firmware. The proposed method provides a way to overwrite the default config so that we don't  have to re-build firmware each time when a new config is needed.